tests/examples_methods_syntax/lasagna_plot.py

```diff
@@ -23,8 +23,8 @@
             format="%Y",
             labelAngle=0,
             labelOverlap=False,
-            labelColor=color_condition,
-            tickColor=color_condition,
+            labelColor=color_condition, # type: ignore[call-overload]
+            tickColor=color_condition, # pyright: ignore[reportArgumentType]
```

Comment:
Leaving these ignore comments in for reference.

Ideally fix the issue that requires these for a `TypedDict` **before** merging this PR.

Reply:
The solve will be using `Map` instead of `dict` in the property setter annotations

Reply:
Will be fixed by #3573